utils/bloom/optimal.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bloom
+
+import (
+	bloomfilter "github.com/holiman/bloomfilter/v2"
+
+	"github.com/ava-labs/avalanchego/utils/math"
+)
+
+// OptimalParameters calculates the optimal [numSeeds] and [numBytes] that
+// should be allocated for a bloom filter which will contain [maxEntries] and
+// target [falsePositiveProbability].
+func OptimalParameters(maxEntries uint64, falsePositiveProbability float64) (int, int) {
+	optimalNumBits := bloomfilter.OptimalM(maxEntries, falsePositiveProbability)
```

Comment:
Is there a reason we don't just implement optimalM/optimalK calculations in this repo like was done for falsePositiveRate?